### ISSUE TITLE
New Version of InuitCSS

### ISSUE THREAD

**Issue by zerodburn** [terabytenz/inuit.css-kitchensink#9]
Hey,

I just thought you should edit your Kitchen Sink to let people know there is a new InuitCSS alpha version out, maybe? https://github.com/inuitcss

Great work on the docs!

Cheers,

Patrick

**Comment by patocallaghan** [comment]
@lindsayjopson what version are you guys using internally these days? Do you think it makes sense to update the docs to the alpha? 

We could always version the docs, have the alpha and v5 branch side-by-side

**Comment by lindsayjopson** [comment]
yeah still using alpha. will check out the new version though and consider an update.

**Comment by HammyNZ** [comment]
I've been using Inuit 5, tho not yet using namespaces at the front eg .o-, .c- etc ( but this makes sense for team /large projects )

Overall it's a relative easy transition, just trying to sort out using npm/bower and being able reference them as alternative paths (eg sass @import will check bower/node_modules directories if no file exists in relative path)

The new version of Inuit class names may have an impact on the base components library (eg .grid is now .layout/.o-layout, .three-quarters now .u-4/3 etc) that needs to be planned.

**Comment by patocallaghan** [comment]
I like the way the EmberJS docs have the version on the left hand side. [MoutJS](http://moutjs.com/) also does doc versioning. I don't think it would be too much of an issue having something like that here. It also wouldn't require Terabyte to upgrade to the latest and greatest internally if it didn't want to.

![inuit](https://cloud.githubusercontent.com/assets/685034/8495749/d84198b0-2167-11e5-9864-9ef96bc91ee2.gif)

Let us know anyway, I wouldn't mind giving a dig out.

**Comment by HammyNZ** [comment]
I'm keen to help out... But where do we start?

On a side note: includePaths for the win using grunt-sass: http://anotheruiguy.gitbooks.io/nodeexpreslibsass_from-scratch/content/bower-grunt-sass.html
(tho I'm trying to figure out how to use importer: https://github.com/sass/node-sass#options - but that can be later)
